Dockerfile.dapper

```diff
@@ -113,11 +113,12 @@ RUN mkdir integration/
 COPY integration/setup.py integration/tox.ini integration/requirements.txt integration/flake8-requirements.txt integration/
 RUN cd integration && tox --notest
 
-# Build longhorn-instance-manager
+# Build longhorn-instance-manager for integraiton testing
 RUN cd /go/src/github.com/longhorn && \
     git clone https://github.com/longhorn/longhorn-instance-manager.git && \
     cd longhorn-instance-manager && \
-    git checkout v1_20201021 && \
+    im_release=$(curl -sfSL https://api.github.com/repos/longhorn/longhorn-instance-manager/releases/latest | jq -r '.tag_name') && \
```

Comment:
We should pin down a version instead of using the latest. An engine in the stable branch might be incompatible with the latest version of instance manager.